h/resources.py

```diff
@@ -86,6 +86,12 @@ def __acl__(self):
 
         return acl
 
+    @staticmethod
```

Comment:
This is a nice pattern. I for one am down with having access bits in the resource classes, even if it bulks them up, as it feels like a natural place for it.

Reply:
Yep, and this is how Pyramid intended it too: this is where access stuff goes in the Pyramid docs and examples etc. Views declare (in `@view_config`) what permissions they require, and if a request doesn't have permission for a view the view never gets called.